src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultGetEventTriggerOperation.java

```diff
@@ -56,9 +66,9 @@ public CaseEventTrigger executeForCaseType(String uid,
                                                                  caseTypeId,
                                                                  eventTriggerId,
                                                                  ignoreWarning),
-                                                                 caseTypeId,
-                                                                 eventTriggerId,
-                                                                 null);
+                     caseTypeId,
```

Comment:
Same comment as above, the code is starting to look like a Picasso painting :)

Reply:
Done